orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -120,9 +120,24 @@ def test_min(self):
 
     def test_equal(self):
         device = self.get_device()
-        cpu_x = torch.ones(3, 3, dtype=torch.float32)
-        cpu_y = torch.ones(3, 3, dtype=torch.float32)
-        assert torch.equal(cpu_x.to(device), cpu_y.to(device))
+        cpu_a = torch.Tensor([1.0, 1.5])
+        ort_a = cpu_a.to(device)
+        cpu_b = torch.Tensor([1.0, 1.5])
+        ort_b = cpu_b.to(device)
+        cpu_c = torch.Tensor([1.0, 1.8])
+        ort_c = cpu_c.to(device)
+        cpu_d = torch.Tensor([1.0, 1.5, 2.1])
+        ort_d = cpu_d.to(device)
+        cpu_e = torch.Tensor([[1.0, 1.5]])
+        ort_e = cpu_e.to(device)
+        assert torch.equal(cpu_a, cpu_b)
```

Comment:
suggestion: did you want to include comment about what aspects / properties each of the different tests are excercising?